features/Skylighting/Shaders/Skylighting/SkylightingCS.hlsl

```diff
@@ -45,130 +47,99 @@ half GetBlueNoise(half2 uv)
 	return BlueNoise[uint3(uv % 128, FrameCount % 64)];
 }
 
-half GetScreenDepth(half depth)
-{
-	return (CameraData.w / (-depth * CameraData.z + CameraData.x));
-}
-
-#define PI 3.1415927
-
 #if !defined(SHADOWMAP)
 [numthreads(8, 8, 1)] void main(uint3 globalId
 								: SV_DispatchThreadID) {
-	float2 uv = float2(globalId.xy + 0.5) * BufferDim.zw * DynamicResolutionParams2.xy;
+	float2 uv = float2(globalId.xy + 0.5) * (BufferDim.zw) * DynamicResolutionParams2.xy;
 	uint eyeIndex = GetEyeIndexFromTexCoord(uv);
 	uv = ConvertFromStereoUV(uv, eyeIndex);
 
-	half3 normalGlossiness = NormalRoughnessTexture[globalId.xy];
-	half3 normalVS = DecodeNormal(normalGlossiness.xy);
-	half3 normalWS = normalize(mul(CameraViewInverse[eyeIndex], half4(normalVS, 0)));
-	half roughness = 1.0 - normalGlossiness.z;
-
 	float rawDepth = DepthTexture[globalId.xy];
 
 	float4 positionCS = float4(2 * float2(uv.x, -uv.y + 1) - 1, rawDepth, 1);
 
 	float4 positionMS = mul(CameraViewProjInverse[eyeIndex], positionCS);
 	positionMS.xyz = positionMS.xyz / positionMS.w;
 
+	positionMS.xyz = positionMS.xyz + CameraPosAdjust[eyeIndex] - EyePosition;
+
 	float3 startPositionMS = positionMS;
 
-	half noise = GetBlueNoise(globalId.xy) * 2.0 * PI;
+	half noise = GetBlueNoise(globalId.xy) * 2.0 * shPI;
 
 	half2x2 rotationMatrix = half2x2(cos(noise), sin(noise), -sin(noise), cos(noise));
 
-	half2 PoissonDisk[16] = {
-		half2(-0.94201624, -0.39906216),
-		half2(0.94558609, -0.76890725),
-		half2(-0.094184101, -0.92938870),
-		half2(0.34495938, 0.29387760),
-		half2(-0.91588581, 0.45771432),
-		half2(-0.81544232, -0.87912464),
-		half2(-0.38277543, 0.27676845),
-		half2(0.97484398, 0.75648379),
-		half2(0.44323325, -0.97511554),
-		half2(0.53742981, -0.47373420),
-		half2(-0.26496911, -0.41893023),
-		half2(0.79197514, 0.19090188),
-		half2(-0.24188840, 0.99706507),
-		half2(-0.81409955, 0.91437590),
-		half2(0.19984126, 0.78641367),
-		half2(0.14383161, -0.14100790)
+	float2 PoissonDisc[] = {
```

Comment:
Where are these values from?

Reply:
https://www.asawicki.info/download.php5?x=Productions%2FApplications%2FPoissonDiscGenerator

Reply:
Whoops. I think we can look into this after the merge.

Reply:
also, can probably put the [-1,1] values in the array rather than remapping it in code.